docs: Add remote config for iOS autocapture settings (#670)
amplitude/amplitude-docs: content/collections/ios_sdk/en/ios-swift-sdk.md

| `optOut`                       | Opt the user out of tracking.                                                                                                                                                                               | `false`                                  |
| ~`defaultTracking`~ (Deprecated. Use [`autocapture`](#autocapture) instead.)             | Enable tracking of default events for sessions, app lifecycles, screen views, and deep links.                                                                                    | `DefaultTrackingOptions(sessions: true)` |
| `autocapture`             | Enable tracking of [Autocapture events](#autocapture) for sessions, app lifecycles, screen views, deep links, network requests, and element interactions.                                                                                    | `AutocaptureOptions.sessions` |

> | `enableAutoCaptureRemoteConfig` | Enable remote configuration for autocapture settings. When enabled, autocapture settings can be updated remotely after initialization.                                                                      | `true` |

Reviewer: ```suggestion
| `enableAutoCaptureRemoteConfig` | Enable remote configuration for autocapture settings. When enabled, Autocapture settings are updateable remotely after initialization.                                                                      | `true` |
```

{{/partial:tabs}}

### Remote configuration for autocapture

Starting from release v1.10.0, the SDK supports remote configuration for autocapture settings. This feature allows you to update autocapture settings after initialization without requiring an app update.

By default, remote configuration for autocapture is enabled. You can disable it by setting `enableAutoCaptureRemoteConfig` to `false` in the configuration:

{{partial:tabs tabs="Swift, Obj-C"}}
{{partial:tab name="Swift"}}
```swift
let amplitude = Amplitude(configuration: Configuration(
    apiKey: "API_KEY",
    autocapture: [.sessions, .appLifecycles, .screenViews],
    enableAutoCaptureRemoteConfig: false
))
```
{{/partial:tab}}
{{partial:tab name="Obj-C"}}
```objc
AMPConfiguration *configuration = [AMPConfiguration initWithApiKey:@"API_KEY"];
configuration.autocapture = [[AMPAutocaptureOptions alloc] initWithOptionsToUnion:@[
    AMPAutocaptureOptions.sessions,
    AMPAutocaptureOptions.appLifecycles,
    AMPAutocaptureOptions.screenViews
]];
configuration.enableAutoCaptureRemoteConfig = NO;
Amplitude *amplitude = [Amplitude initWithConfiguration:configuration];
```
{{/partial:tab}}
{{/partial:tabs}}

> When remote configuration is enabled, the SDK will check for updates to autocapture settings from the Amplitude servers. If updates are available, the SDK will apply them automatically. This allows you to:

Reviewer: ```suggestion
When you enable remote configuration, the SDK checks for updates to Autocapture settings in your project. If updates are available, the SDK applies them automatically. This allows you to:
```
